spec/acceptance/firewall_attributes_ipv6_exceptions_spec.rb

```diff
@@ -86,29 +86,6 @@ class { '::firewall': }
             end
           end
         end
-
-        context 'when multiple addrtype fail', if: (os[:family] == 'redhat' && os[:release].start_with?('5')) do
-          pp = <<-PUPPETCODE
-                class { '::firewall': }
-                firewall { '616 - test':
-                  proto    => tcp,
-                  jump     => accept,
-                  #{type}  => ['LOCAL', '! LOCAL'],
-                  protocol => 'IPv6',
-                }
-          PUPPETCODE
-          it 'fails' do
-            apply_manifest(pp, expect_failures: true) do |r|
-              expect(r.stderr).to match(%r{Multiple #{type} elements are available from iptables version})
-            end
-          end
-
-          it 'does not contain the rule' do
-            run_shell('ip6tables-save') do |r|
-              expect(r.stdout).not_to match(%r{-A INPUT -p (tcp|6) -m addrtype --#{type.tr('_', '-')} LOCAL -m addrtype ! --#{type.tr('_', '-')} LOCAL -m comment --comment "616 - test" -j ACCEPT})
-            end
-          end
-        end
```

Comment:
at line 120 the line seems to be problematic :
```
describe 'ipset', if: (os[:family] == 'redhat' && os[:release].start_with?('14')) do
```
it should be I think : 
```
describe 'ipset', if: (os[:family] == 'debian' && os[:release].start_with?('14')) do
```

Reply:
Added debian.